roles/fstat/tasks/main.yml

```diff
@@ -1,17 +1,35 @@
 ---
+- user: name={{user}}
+  sudo: yes
+
 - name: Install required packages
-  package:
+  yum:
     state: present
     name: "{{ item }}"
   with_items:
-  - python-virtualenv
-  - gcc
-  - git
+    - python-virtualenv
+    - gcc
+    - git
 
-# TODO create the directory /fstat, do the git checkout
-# automate update
-# set configuration file, and database creation
+# TODO set configuration file, and database creation
 # and selinux boolean
+
+- name: create fstat directory
+  file: path=/fstat/code state=directory
+
+- name: create virtualenv if it does not exists
+  command: virtualenv /fstat/venv
```

Comment:
This command will be run each time, I think that's suboptimal, shouldn't it verify that ?